public/v4/apps/appwrite.yml

```diff
@@ -0,0 +1,1338 @@
+            _APP_WORKER_PER_CORE: $$cap_APP_WORKER_PER_CORE
+            _APP_REDIS_HOST: srv-captain--$$cap_appname-redis
+            _APP_REDIS_PORT: 6379
+            _APP_REDIS_USER:
+            _APP_REDIS_PASS:
+            _APP_DB_HOST: srv-captain--$$cap_appname-mariadb
+            _APP_DB_PORT: 3306
+            _APP_DB_SCHEMA: $$cap_APP_DB_SCHEMA
+            _APP_DB_USER: $$cap_APP_DB_USER
+            _APP_DB_PASS: $$cap_APP_DB_PASS
+            _APP_DB_ROOT_PASS: $$cap_APP_DB_ROOT_PASS
+            _APP_INFLUXDB_HOST: $$cap_appname-influxdb
+            _APP_INFLUXDB_PORT: 8086
+            _APP_STATSD_HOST: srv-captain--$$cap_appname-telegraf
+            _APP_STATSD_PORT: 8125
+            _APP_SMTP_HOST: $$cap_APP_SMTP_HOST
+            _APP_SMTP_PORT: $$cap_APP_SMTP_PORT
+            _APP_SMTP_SECURE: $$cap_APP_SMTP_SECURE
+            _APP_SMTP_USERNAME: $$cap_APP_SMTP_USERNAME
+            _APP_SMTP_PASSWORD: $$cap_APP_SMTP_PASSWORD
+            _APP_STORAGE_LIMIT: $$cap_APP_STORAGE_LIMIT
+            _APP_STORAGE_PREVIEW_LIMIT: $$cap_APP_STORAGE_PREVIEW_LIMIT
+            _APP_STORAGE_DEVICE: $$cap_APP_STORAGE_DEVICE
+            _APP_STORAGE_ANTIVIRUS: disabled
+            _APP_STORAGE_ANTIVIRUS_HOST: clamav
+            _APP_STORAGE_ANTIVIRUS_PORT: 3310
+            _APP_STORAGE_S3_ACCESS_KEY:
+            _APP_STORAGE_S3_SECRET:
+            _APP_STORAGE_S3_REGION:
+            _APP_STORAGE_S3_BUCKET:
+            _APP_STORAGE_DO_SPACES_ACCESS_KEY:
+            _APP_STORAGE_DO_SPACES_SECRET:
+            _APP_STORAGE_DO_SPACES_REGION:
+            _APP_STORAGE_DO_SPACES_BUCKET:
+            _APP_FUNCTIONS_SIZE_LIMIT: $$cap_APP_FUNCTIONS_SIZE_LIMIT
+            _APP_FUNCTIONS_TIMEOUT: $$cap_APP_FUNCTIONS_TIMEOUT
+            _APP_FUNCTIONS_BUILD_TIMEOUT: $$cap_APP_FUNCTIONS_BUILD_TIMEOUT
+            _APP_FUNCTIONS_CONTAINERS: $$cap_APP_FUNCTIONS_CONTAINERS
+            _APP_FUNCTIONS_CPUS: $$cap_APP_FUNCTIONS_CPUS
+            _APP_FUNCTIONS_MEMORY: $$cap_APP_FUNCTIONS_MEMORY
+            _APP_FUNCTIONS_MEMORY_SWAP: $$cap_APP_FUNCTIONS_MEMORY_SWAP
+            _APP_FUNCTIONS_RUNTIMES: $$cap_APP_FUNCTIONS_RUNTIMES
+            _APP_EXECUTOR_SECRET: $$cap_APP_EXECUTOR_SECRET
+            _APP_EXECUTOR_RUNTIME_NETWORK: srv-captain--$$cap_appname-runtimes
+            _APP_FUNCTIONS_ENVS: node-16.0,php-7.4,python-3.9,ruby-3.0
+            _APP_FUNCTIONS_INACTIVE_THRESHOLD: $$cap_APP_FUNCTIONS_INACTIVE_THRESHOLD
+            _APP_MAINTENANCE_INTERVAL:
+            _APP_MAINTENANCE_RETENTION_EXECUTION:
+            _APP_MAINTENANCE_RETENTION_ABUSE:
+            _APP_MAINTENANCE_RETENTION_AUDIT:
+            DOCKERHUB_PULL_USERNAME: $$cap_DOCKERHUB_PULL_USERNAME
+            DOCKERHUB_PULL_PASSWORD: $$cap_DOCKERHUB_PULL_PASSWORD
+            DOCKERHUB_PULL_EMAIL: $$cap_DOCKERHUB_PULL_EMAIL
+
+    $$cap_appname-appwrite-realtime:
+        image: 'appwrite/appwrite:$$cap_APP_VERSION'
+        entrypoint: realtime
```

Comment:
Similar to `command`, `entrypoint` is not consumed by CapRover. Please read docs and if needed convert to `dockerfileLines`